### PR TITLE
Multiple fixes for Bert Model (fine-tuned) 

### PR DESCRIPTION
**Pull request by jignparm**
Allow for dynamic output shapes in Onnx model [ i.e. rank=1, shape =(?,)] 
Fix 'Squeeze' op to handle shape of (?,)
Refactor 'Constant_folding'  rewriter to pass unit tests

### CODE REVIEW

File: tf2onnx/tfonnx.py [hunk 1]
```diff
@@ -54,28 +54,26 @@ def rewrite_constant_fold(g, ops):
...
         keep_looking = False
         for idx, op in enumerate(ops):
             func = func_map.get(op.type)
-            if func is None:
-                continue
+            if func is None: continue
+            if set(op.output) & set(g.outputs): continue
             try:
                 inputs = []
+                skip = False
```
Comment (guschmue) [inline tf2onnx/tfonnx.py:68]: need the skip? we already check that all inputs are const below

File: tf2onnx/tfonnx.py [hunk 2]
```diff
@@ -466,7 +460,7 @@ def compat_handler(ctx, node, **kwargs):
                  rewrite_single_direction_lstm, rewrite_bi_direction_lstm,
                  rewrite_single_direction_gru, rewrite_bi_direction_gru,
                  rewrite_custom_rnn_cell, rewrite_generic_loop, rewrite_cond,
-                 rewrite_biasadd_with_conv2d,
+                 rewrite_biasadd_with_conv2d, rewrite_constant_fold
```
Comment (guschmue) [inline tf2onnx/tfonnx.py:463]: might help to run const_cold first in the longer term but maybe more risk right now

File: tf2onnx/tfonnx.py (hunk 1 above)
Comment (jignparm) [reply tf2onnx/tfonnx.py]: Thanks for the catch!  `inputs and len(op.input) == len(inputs)` will only run if all are `const`. Removed the `skip`

File: tf2onnx/tfonnx.py (hunk 2 above)
Comment (jignparm) [reply tf2onnx/tfonnx.py]: Moved it to first rewriter.

**Pull request merged**